### PR TITLE
ingested iwslt 2016

### PR DESCRIPTION
**Pull request by davidstap**
See [here](https://workshop2016.iwslt.org/index.php) for original website. The proceedings is not on the website, and was sent to us via email. 

* No full proceedings PDF available.
* No page numbers available.

### CODE REVIEW

**Comment by davidstap** [comment]
small bibfix issue - I’ll fix this later

**Comment by mjpost** [comment]
The problem is a duplicate bibkey in 2016.iwslt.xml. However, running the bibkey script after deleting it doesn't generate any new key for it (even with -c). I'm not sure why and ran out of time to investigate. Pinging @mbollmann in case he has time to take a look.

**Comment by mjpost** [comment]
It seems the default value for a paper's bibkey was set to `False` instead of `None`: in which case I'm not sure why this worked at all! I changed this, we'll see if it causes a problem elsewhere.

File: data/xml/2016.iwslt.xml [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+      <url hash="79d08aed">2016.iwslt-1.10</url>
+      <bibkey>cho-etal-2016-multilingual</bibkey>
+    </paper>
+    <paper id="11">
+      <title>A Neural Verb Lexicon Model with Source-side Syntactic Context for String-to-Tree Machine Translation</title>
+      <author><first>Maria</first><last>Nadejde</last></author>
```
Comment (mjpost) [inline data/xml/2016.iwslt.xml:116]: Maria Nădejde on the PDF

File: data/xml/2016.iwslt.xml [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+      <author><first>Tom</first><last>Kocmi</last></author>
+      <author><first>Roman</first><last>Sudarikov</last></author>
+      <abstract>We present our submissions to the IWSLT 2016 machine translation task, as our first attempt to translate subtitles and one of our early experiments with neural machine translation (NMT). We focus primarily on English→Czech translation direction but perform also basic adaptation experiments for NMT with German and also the reverse direction. Three MT systems are tested: (1) our Chimera, a tight combination of phrase-based MT and deep linguistic processing, (2) Neural Monkey, our implementation of a NMT system in TensorFlow and (3) Nematus, an established NMT system.</abstract>
+      <url hash="a14058e8">2016.iwslt-1.24</url>
+      <bibkey>bojar-etal-2016-ufal</bibkey>
+    </paper>
+    <paper id="25">
+      <title>The <fixed-case>UMD</fixed-case> Machine Translation Systems at <fixed-case>IWSLT</fixed-case> 2016: <fixed-case>E</fixed-case>nglish-to-<fixed-case>F</fixed-case>rench Translation of Speech Transcripts</title>
+      <author><first>Xing Niu Marine</first><last>Carpuat</last></author>
```
Comment (mjpost) [inline data/xml/2016.iwslt.xml:276]: Two names

**Review by mjpost — COMMENTED** [review]
A few fixes, then LGTM

**Comment by mbollmann** [comment]
> It seems the default value for a paper's bibkey was set to `False` instead of `None`: in which case I'm not sure why this worked at all! I changed this, we'll see if it causes a problem elsewhere.

Looks like I explicitly changed that in the "preformatted citation formats" branch, so there might have been a reason for that. I can't look into that today, maybe hold off with merging this a little bit?

**Comment by mbollmann** [comment]
> It seems the default value for a paper's bibkey was set to `False` instead of `None`: in which case I'm not sure why this worked at all! I changed this, we'll see if it causes a problem elsewhere.

[git blame says](https://github.com/acl-org/acl-anthology/blame/8df7bdb9ea211ccb39aa8d1a6b006d1fcb18f7fb/bin/anthology/papers.py) that this change originated from a 3 years old commit:

git blame showing the origin of this line

I don't know how that ended up in this pull request; maybe I have resolved a merge conflict wrong at some point.

Sorry for breaking things & thanks for catching (the cause of) this!

**Review by mjpost — APPROVED** [review]

**Pull request merged**